pkgs/data/misc/mobile-broadband-provider-info/default.nix

```diff
@@ -1,17 +1,28 @@
-{ lib, stdenv, fetchurl, gnome, libxslt }:
+{ lib
+, stdenv
+, fetchurl
+, gnome
+, libxslt
+, libxml2
+, meson
+, ninja
+, pkg-config
```

Comment:
Also here. And since you are reformatting, might be a good idea to use the nixfmt-rfc-style.

Reply:
Dropped `, pkg-config` and ran the formatter.